src/components/UI/FormTextArea.vue

```diff
@@ -0,0 +1,80 @@
+<script lang="ts" setup>
+import AutoResizeTextArea from '/@/components/UI/AutoResizeTextArea.vue'
+import { computed } from 'vue'
+
+interface Props {
+  modelValue: string
+  placeholder?: string
+  limit?: number
+  rows?: number
+  maxHeight?: number
+  name?: string
+}
+
+const props = withDefaults(defineProps<Props>(), {
+  placeholder: '',
+  limit: undefined,
+  rows: 1,
+  maxHeight: undefined,
+  name: undefined
+})
+
+const emit = defineEmits<{
+  (e: 'update:modelValue', modelValue: string): void
+}>()
+
+const value = computed({
+  get: () => props.modelValue,
+  set: v => emit('update:modelValue', v)
+})
+
+const isExceeded = computed(
+  () => props.limit && [...props.modelValue].length > props.limit
+)
+</script>
+
+<template>
+  <div :class="$style.container">
+    <auto-resize-text-area
+      v-model="value"
+      :placeholder="props.placeholder"
+      :class="$style.textarea"
+      :row="props.rows"
```

Comment:
```suggestion
      :rows="props.rows"
```

多分typoだと思うのですが、直してもrowsが機能してなかったので多分実装してもらったオートリサイズ機能に妨げられてる気がします🤔